voter-dapp/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "UMA Voter DApp",
+  "name": "UMA Voter DApp",
+  "icons": [
+    {
+      "src": "favicon.ico",
```

Comment:
nit: did you swap out the favicon for the UMA one that's in the `sponsor-dapp` directory?

Reply:
Yeah I copied it over from there.

$ diff ../sponsor-dapp/public/favicon.ico public/favicon.ico

Reply:
Sweet. Just wanted to double check.